library/SmartyPlugins/function.follow_button.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Returns category follow toggle button.
+ *
+ * @copyright 2009-2019 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ * @package vanilla-smarty
+ * @since 2.0
+ * @return string
+ */
+
+/**
+ * Follow button.
+ *
+ * @return string
+ */
+function smarty_function_follow_button() {
+    $followButton = '';
+    $controller = Gdn::controller();
+    $controller->fetchViewLocation('helper_functions', 'categories', false, false);
```

Comment:
I see you've fetched the view location but are not including it. The function happens to be included from somewhere else but that may not always be the case. Be sure to do a `require_once` of the actual view location.